images/linux/scripts/installers/haskell.sh

```diff
@@ -53,10 +55,11 @@ if ! command -v stack; then
     exit 1
 fi
 
+# Add the latest ghc to path
+prependEtcEnvironmentPath "/opt/ghc/$defaultGHCVersion/bin"
+
 # Document what was added to the image
 echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "Haskell Cabal ($(/opt/cabal/$cabalVersion/bin/cabal --version))"
-for version in ${ghcInstalledVersions[@]}; do
-    DocumentInstalledItem "GHC ($(/opt/ghc/$version/bin/ghc --version))"
-done
+DocumentInstalledItem "GHC ($(/opt/ghc/$defaultGHCVersion/bin/ghc --version))"
```

Comment:
If we put ghc to PATH and remove other versions, why we need to print path here?
I just worry that this PATH will be changed on every version release

Reply:
@maxim-lobanov , the path to default ghc will not be shown on software report, it is just invoke ghc to take version. On software report it looks like:
```
- GHC (The Glorious Glasgow Haskell Compilation System, version 8.10.1)
```

Reply:
Oh, sorry, I missed it